packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -313,12 +317,19 @@ export class PreferencesEditorsContainer extends DockPanel {
 
         super.onAfterAttach(msg);
         this.onInitEmitter.fire(undefined);
+        this.toDispose.push(
```

Comment:
It should be toDisposeOnDetach otherwise a new listener will be added on each drag and drop